src/config/index.ts

```diff
@@ -90,8 +90,12 @@ export enum BuildStatus {
  * GitHub Triage
  */
 export const PRODUCT_AREA_LABEL_PREFIX = 'Product Area: ';
+export const STATUS_LABEL_PREFIX = 'Status: ';
+export const BACKLOG_LABEL = 'Status: Backlog';
+export const IN_PROGRESS_LABEL = 'Status: In Progress';
 export const UNTRIAGED_LABEL = 'Status: Untriaged';
 export const UNROUTED_LABEL = 'Status: Unrouted';
+export const UNKNOWN_LABEL = 'Status: Unknown';
```

Comment:
Now my brain wants to abstrack these even more with `'Status: ${whatever}'` but I'll stop myself there 😅